connectors/src/connectors/microsoft/temporal/workflows.ts

```diff
@@ -121,3 +137,7 @@ export function microsoftFullSyncSitesWorkflowId(connectorId: ModelId) {
 export function microsoftIncrementalSyncWorkflowId(connectorId: ModelId) {
   return `microsoft-incrementalSync-${connectorId}`;
 }
+
+export function microsoftGarbageCollectorWorkflowId(connectorId: ModelId) {
```

Comment:
here we should be able to have mutliple garbage collection workflow per connector
maybe a hash of the nodes would do?